packages/shared/src/UI/components/AssetPlayer/index.tsx

```diff
@@ -69,28 +64,34 @@ export const AssetPlayer = memo<AssetPlayerProps>(({ url, type, options, iconPro
     //#region setup iframe when url and options be changed
     const setIframe = useCallback(() => {
         // if iframe isn't be init or the load error has been existed
-        if (!ref.current || playerState === AssetPlayerState.ERROR) return
-        if (!url) {
+        if (!ref.current || playerState === AssetPlayerState.ERROR || playerState === AssetPlayerState.NORMAL) return
```

Comment:
Reduce sending repeat requests, which causes the image flashes multiple times. 

cc @nuanyang233

Reply:
This does not allow for resending the message after the prop is changed to make the iframe render the new asset. But it does not affect the current.